actions/scoring.js

```diff
@@ -131,27 +143,101 @@ async function getNFTs(address){
 async function getZapperNFT(address) {}
 
 //###################### TOTAL SCORE #############################
-export async function computeScore(address, chainId) {
-  let score = BASE_SCORE;
+
+async function getRawValues(address, chainId) {
   //get token list
   const tokenList = await getAllTokenBalances(address, chainId);
-  //get scores
+  //get values
+  const USDBalance = await getUSDBalance(tokenList);
+  const tokenHoldingsRaw = await getQuantityOfTokens(tokenList);
+
   const scamTokenScore = await getScamTokens(tokenList);
   const govTokenScore = await getGovernanceTokens(tokenList);
   const aaveGovScore = await getAaveGovernanceVotes(address);
   const compInteractionScore = await getCompoundInteractions(address, chainId);
-  // let ensScore = await hasENS(address);
   const NFTScore = await getNFTs(address)
 
-  // let ensScore = await hasENS(address);
-  score += -scamTokenScore + govTokenScore + aaveGovScore + compInteractionScore + NFTScore;
+  return {
+      usd_balance: USDBalance,
+      token_holdings_raw: tokenHoldingsRaw,
+      scam_raw: scamTokenScore,
+      governance_raw: govTokenScore,
+      aave_votes_raw: aaveGovScore,
+      compound_interactions_raw: compInteractionScore,
+      NFT_score_raw: NFTScore,
+  }
+}
+
+async function computeScoreFromRaw(rawValues) {
+  let score = BASE_SCORE;
+  //get scores
+  let USDScore = 0;
+  if(rawValues.usd_balance <= 100){
+    USDScore = 10;
+  }
+  else if(rawValues.usd_balance > 100 && rawValues.usd_balance <= 1000){
+    USDScore = 20
+  }
+  else if(rawValues.usd_balance > 1000 && rawValues.usd_balance <= 10000){
+    USDScore = 50
+  }
+  else if(rawValues.usd_balance > 10000){
```

Comment:
Could be replaced by an `else` statement.

Reply:
Isn't it better to display all ranges ?